go.mod

```diff
@@ -81,7 +81,6 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
```

Comment:
Is the `go.mod` and `go.sum` change in expect?

Reply:
Yes, I do it with `rm go.sum && go mod tidy` is same as now.